commands/createfixpullrequests_test.go

```diff
@@ -158,12 +171,16 @@ func TestCreateFixPullRequestsCmd_Run(t *testing.T) {
 			err = cmd.Run(configAggregator, client)
 			// Validate
 			assert.NoError(t, err)
-			resultDiff, err := verifyDependencyFileDiff("main", test.expectedBranchName, test.dependencyFileName)
-			assert.NoError(t, err)
-			assert.Equal(t, test.expectedDiff, string(resultDiff))
+			for _, branch := range test.expectedBranches {
+				resultDiff, err := verifyDependencyFileDiff("main", branch, test.packageDescriptorPaths...)
+				assert.NoError(t, err)
+				assert.Contains(t, test.expectedDiff, string(resultDiff))
+			}
 			// Defers
-			restoreEnv()
-			server.Close()
+			defer func() {
+				restoreEnv()
+				server.Close()
```

Comment:
```suggestion
				assert.NoError(server.Close())
```
and same for all err func on this file

Reply:
defer cleanUp() >>

defer func(){
    assert.NoError(clenaup())
}

Reply:
these function does not return errors